client/reader/stream/post.jsx

```diff
@@ -363,7 +363,7 @@ const Post = React.createClass( {
 				{ shouldUseFullExcerpt
 					? <EmbedContainer>
 							<div key="full-post-inline"
-								className="reader__full-post-content"
+								className="reader-full-post__story-content"
```

Comment:
I know the stream shouldn't be using this class, but historically it has shared content styles with the full post view. We'll phase this out when we launch the refreshed stream cards later this year.